### PR TITLE
Created the signup page and login page ui

### LINKED ISSUE #162: [FEATURE] Build sign up page
**Description**

When a user clicks the sign-up button, a sign-up page must occur, maybe authentication will be added in the future. Please assign me to work on this.

### PR DESCRIPTION
**Pull request by aasimtaif**
**What does this PR do?**
I have created a responsive signup and Login page. It is just UI that takes the name, email & password  with no authentication functionality, If you want  authentication  I can do it 

**Related Issue(s)**
fixes #162 

**Changes**
Now If you click on the signup button on the home page it will take you to the signup page 

**Screenshots or GIFs (if applicable)**
Screenshot from 2023-10-23 22-10-17
Screenshot from 2023-10-27 20-42-40

**Checklist**
- [x] I have read and followed the [contributing guidelines](../CONTRIBUTING.md).
- [x] My code follows the project's coding style and conventions.
- [ ] I have added unit tests (if applicable) to ensure the code works as expected.
- [ ] I have updated the project documentation (if necessary).

**Additional Comments**
Add any additional comments, context, or information about this PR.

**By submitting this PR, I confirm that:**
- [x] I have reviewed my code and believe it is ready for merging.
- [x] I understand that this PR may be subject to review and changes.
- [x] I agree to abide by the code of conduct and contributing guidelines of this project.

### CODE REVIEW

**Comment by Vijaykv5** [comment]
@aasimtaif as the whole project is implemented in typescript, Please move your file to typescript

i.e, .jsx -> /tsx

**Comment by aasimtaif** [comment]
> @aasimtaif as the whole project is implemented in typescript, Please move your file to typescript
> 
> i.e, .jsx -> /tsx

ok

File: src/Pages/SignUpPage.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                            </div>
+                            <button type="submit" className="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 mr-2 mb-2 dark:bg-blue-600 dark:hover:bg-blue-700 focus:outline-none dark:focus:ring-blue-800
+                            w-full ">Create an account</button>
+                            <p className="text-sm font-light text-gray-400">
+                                Already have an account? <Link to="/login" className="font-medium hover:underline text-primary-500">
+                                    Login here
+                                </Link>
+                            </p>
+                        </form>
+                    </div>
```
Comment (Vijaykv5) [inline src/Pages/SignUpPage.tsx:41]: change color of button to violet as per website theme

Comment (aasimtaif) [reply src/Pages/SignUpPage.tsx:41]: ok

File: src/index.tsx [hunk 2]
```diff
@@ -11,11 +12,14 @@ import { Route, Routes } from "react-router-dom";
...
-    <Routes>
-    <Route path="/" element={<HomePage />} />
-    <Route path="/search" element={<MainSection />} />
-    <Route path="/docs" element={<DocsPage />}/>
-      
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/search" element={<MainSection />} />
+        <Route path="/docs" element={<DocsPage />} />
+        <Route path="/signUp" element={<SignUpPage />} />
```
Comment (Vijaykv5) [inline src/index.tsx:19]: Please change the path to "/signup"

**Comment by Vijaykv5** [comment]
May you also fix the above thing and resolve the conflicts?

After that I'll merge this PR :)

**Comment by aasimtaif** [comment]
If don't you merge quickly there will me more conflicts

**Comment by Vijaykv5** [comment]
Please fix the above changes I reviewed 
Soon after that I'll merge

**Comment by aasimtaif** [comment]
> Please fix the above changes I reviewed Soon after that I'll merge

I did

File: src/index.tsx (hunk 2 above)
Comment (Vijaykv5) [reply src/index.tsx:19]: This one

**Comment by Vijaykv5** [comment]
Screenshot 2023-10-29 at 9 03 43 PM

Not reflected "signup" and not "signUp"

File: src/index.tsx (hunk 2 above)
Comment (aasimtaif) [reply src/index.tsx:19]: ok

**Comment by aasimtaif** [comment]
If you want  anything changed tell me or Merge the PR

**Comment by aasimtaif** [comment]
> Please fix the above changes I reviewed Soon after that I'll merge

???

**Comment by aasimtaif** [comment]
hi @Vijaykv5 I would really appreciate it if you merge the PR before 1 November

**Comment by Vijaykv5** [comment]
@aasimtaif Sure I will merge it..

**Comment by Vijaykv5** [comment]
Thanks for your Contribution 🚀

**Pull request merged**